composer.json

```diff
@@ -24,12 +24,12 @@
     },
     "minimum-stability": "dev",
     "require": {
-        "typo3/cms": ">=7.5.0 <=7.7.99",
-        "typo3-ter/dyncss": ">=0.7.2 <=0.7.99",
-        "typo3-ter/dyncss-less": ">=0.7.0 <=0.7.99",
-        "typo3-themes/themes": ">=7.0.2 <=7.2.99",
+        "typo3/cms": "^7.6",
+        "typo3-ter/dyncss": "0.7.9 ",
```

Comment:
If we want to use a newer version of extension we should update it on composer.json but at the same time keep the possibility to add patches. That's mean we should use range except for cases when we want to freeze dependencies.
So I would suggest to use:
`"typo3-ter/dyncss": ">=0.7.8 <=0.7.99"`